jbrowse/src/client/JBrowse/Browser/plugins/ExtendedVariantPlugin/ExtendedVariantWidget/ExtendedVariantWidget.js

```diff
@@ -31,71 +33,323 @@ export default jbrowse => {
+                )
+            }
+
+            var alleleTableRows = []
+            for(var allele in alleleCounts){
+                alleleTableRows.push(
+                    <TableRow>
+                            <TableCell>{allele}</TableCell>
+                            <TableCell>{round(alleleCounts[allele]/alleleTotal, 4)}</TableCell>
+                            <TableCell>{alleleCounts[allele]}</TableCell>
+                    </TableRow>
+                )
+            }
+            var gtTitle = "Genotype Frequency ("+gtTotal.toString()+")"
+
+            // TODO - get variables, prepare genotypeTable link
+            var trackId = 0
+            var contig = 0
+            var start = 0
+            var end = 0
+            var link = ActionURL.buildURL("jbrowse", "genotypeTable.view", null, {trackId: trackId, chr: contig, start: start, stop: end})
+            var href = <a href={link}>Click here to view sample-level genotypes</a>
+
+            setState(
+            <div>
+                <BaseCard title="Allele Frequencies">
+                    <Table className={classes.table}>
+                        <TableHead>
+                            <TableRow className={classes.paperRoot}>
+                                    <TableCell>Sequence</TableCell>
+                                    <TableCell>Fraction</TableCell>
+                                    <TableCell>Count</TableCell>
+                            </TableRow>
+                        </TableHead>
+                        <TableBody>
+                            {alleleTableRows}
+                        </TableBody>
+                    </Table>
+                </BaseCard>
+                <BaseCard title={gtTitle}>
+                     <Chart
+                       width={'250px'}
+                       height={'200px'}
+                       chartType="BarChart"
+                       loader={<div>Loading Chart</div>}
+                       data={gtBarData}
+                       options={{
+                         title: "Genotypes",
+                         width: 300,
+                         height: 200,
+                         bar: { groupWidth: '95%' },
+                         legend: { position: 'none' },
+                       }}
+                       // For tests
+                       rootProps={{ 'data-testid': '6' }}
+                     />
+                     <div className={classes.link}>
+                        {href}
+                     </div>
+                </BaseCard>
+            </div>)
+        }, []);
+        return state
+    }
     function NewTable(props) {
         const classes = useStyles()
         const { feature } = props
         const { model } = props
         const feat = JSON.parse(JSON.stringify(model.featureData))
-        var displays;
+        const samples = model.featureData.samples
+        console.log(model.featureData)
```

Comment:
console.log here, just in case you didn't want to commit it

Reply:
Removed, nice catch